java-manta-client-unshaded/src/main/java/com/joyent/manta/client/crypto/ExternalSecurityProviderLoader.java

```diff
@@ -48,14 +67,22 @@
      */
     private static final Provider BC_PROVIDER;
 
+    /**
+     * Reference to the Sun JCE Provider.
+     */
+    private static final Provider SUNJCE_PROVIDER;
+
+    /**
+     * A unique list of security provider supported for client-side encryption
+     * in ranked order of preference.
+     */
+    private static List<Provider> rankedPreferredProviders;
+
     static {
-        final Provider pkcs11Provider = Security.getProvider(PKCS11_PROVIDER_NAME);
+        PKCS11_PROVIDER = Security.getProvider(PKCS11_PROVIDER_NAME);
 
-        if (pkcs11Provider != null) {
-            PKCS11_PROVIDER = pkcs11Provider;
+        if (PKCS11_PROVIDER != null) {
             LOGGER.debug("PKCS11 NSS provider was loaded - native crypto support enabled");
```

Comment:
In light of the ranked preference, I'm not sure this message should still say *enabled*.

Reply:
Changed.